documentation/source/users/rmg/input.rst

```diff
@@ -942,3 +942,29 @@ For example ::
 		maxNumSpecies=100
 	)
 
+Restarting from a Seed Mechanism
+=================================
+The preferred method for restarting an RMG-Py job is to restart the job from a seed mechanism. There are many scenarios
+when the user might want to do this, including continuing on a job that ran out of time or crashed as the result of a
+now fixed bug. To restart from a seed mechanism, the block below must be added on to the input file.  ::
+
+    restartFromSeed(path='seed')
+
+The ``path`` flag is the path to the seed mechanism folder, which contains three subfolders that must be titled as
+follows: Filters, Seed, Seed_Edge. The path can be a relative path from the input.py file or an absolute path on disk
```

Comment:
So I appreciate the simplicity of simply passing the path of the directory in a job you ran, however, in general we should have an interface (at least in addition to the current one) that isn't dependent on the directory structure beneath the input path.  The first approach works great until someone manipulates the files or moves the files around and then the path only works if they rearrange them in the exact right directory structure.  Taking each file as a separate input is much more typical of professional python codes.

Reply:
Fixed to allow both options as we discussed offline